apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -3,6 +3,11 @@ import * as drawUtils from '@cdo/apps/p5lab/drawUtils';
 import commands from './commands/index';
 import {APP_HEIGHT, APP_WIDTH} from '../constants';
 
+// Big numbers in some blocks can cause performance issues. Combined with live-preview,
+// this results in hanging the tab and students unable to edit their blocks. We
+// guard against this by capping numbers where needed.
+const SPRITE_BIG_NUMBER_GUARD = 1000;
```

Comment:
What do you think about storing these constants in a separate file so we can share some of these numbers? I see we reference `SPRITE_BIG_NUMBER_GUARD` in two different places and they have the same value. Maybe some sort of `p5lab/spritelab/constants.js` or `limits.js` or something?

Reply:
Great idea! I will place in the existing spritelab/constants.js file.